src/plugins/irc-events/message.js

```diff
@@ -38,19 +38,20 @@ module.exports = function(irc, network) {
 
 	function handleMessage(data) {
 		let chan;
+		const from = chan.getUser(data.nick);
```

Comment:
This breaks the server.

Chan doesn't exist yet.

Reply:
Fuck.